ripple/src/main/webapp/businessdetails.js

```diff
@@ -52,4 +52,172 @@ function removeColorFromRating(ratingType, rating, maxRating, elementIdPrefix, a
 /* TODO: Function will add new review to firestore, then refresh the page to see changes */
 function newReview() {
   location.reload();
+  clearLocalStorage(["starRating", "priceRating"]);
+}
+
+/* Load all business details on the body onload */
+function bdOnload() {
+  // TEMPORARY LOCAL STORAGE UNTIL MANAGE BUSINESS INFO PAGE IS COMPLETE
+  localStorage.setItem("walkingTime", 15); 
```

Comment:
Can we dynamically pull these values since they are all set within local storage when a user clicks a business?

Reply:
We can remove the .setItem calls, basically

Reply:
Yeah, I wrote these functions before you merged your code for setting in local storage. That's what I meant by the comment for temporary local storage (ignore Manage Business info part; see comment above)

Reply:
Ah ok, sounds good.